### PR TITLE
New load and extracts for hail datasets 

### PR DESCRIPTION
**Pull request by GreatBrando**
Added new load and extract files for additional datasets such as GERP++, variant_summary

### CODE REVIEW

File: datasets/load/load.Ensembl_homo_sapiens_low_complexity_regions.py [hunk 1]
```diff
@@ -24,16 +24,13 @@
...
                                             version=f'release_{version}',
                                             reference_genome=build,
-                                            n_rows=n_rows,
                                             n_partitions=n_partitions))
 
 path = f'{hail_data_root}/{name}.release_{version}.{build}.ht'
 ht.write(path, overwrite=True)
-ht = hl.read_table(path)
-ht.describe()
+
```
Comment (danking) [inline datasets/load/load.Ensembl_homo_sapiens_low_complexity_regions.py]: no unnecessary whitespace

File: datasets/load/load.GERP++.py [hunk 2]
```diff
@@ -42,12 +42,11 @@
     ht = ht.select('interval', 'S', 'p_value')
     ht = ht.key_by('interval')
 
-n_rows = ht.count()
+
```
Comment (danking) [inline datasets/load/load.GERP++.py]: no unnecessary blank lines

File: datasets/load/load.gencode_v30_annotation_gff3.py [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
+parser.add_argument('-b', required=True, choices=['GRCh37', 'GRCh38'], help='gencode build.')
+args = parser.parse_args()
 
-t2 = t2.filter(t2.f2 == "gene")
+version = args.v
+build = args.b
 
-t2 = t2.annotate(interval=hl.interval(hl.locus(t2.f0, t2.f3, 'GRCh38'), hl.locus(t2.f0, t2.f3, 'GRCh38')))
+if version == 'v19':
+
```
Comment (danking) [inline datasets/load/load.gencode_v30_annotation_gff3.py]: something seems wrong here

File: hail/python/hail/experimental/db.py [hunk 4]
```diff
@@ -27,7 +27,7 @@ def annotate_rows_db(self,mt,*names):
             
             >>> db = hl.experimental.DB()
             >>> mt = db.annotate_rows_db(mt, 'gnomad_lof_metrics')
-            ...
+            >>> mt = db.annotate_rows_db(mt, 'vep', 'CADD', 'gnomAD')
```
Comment (danking) [inline hail/python/hail/experimental/db.py]: let's make the example have four on one line

**Comment by tpoterba** [comment]
@danking do you want to make the changes you commented so this can get merged?

**Comment by johnc1231** [comment]
@danking What's the status of this one?

**Comment by danking** [comment]
We'll have to return to this later.
